x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/ValidateDetectorAction.java

```diff
@@ -30,14 +28,6 @@ protected ValidateDetectorAction() {
         super(NAME, AcknowledgedResponse::new);
     }
 
-    public static class RequestBuilder extends ActionRequestBuilder<Request, AcknowledgedResponse> {
-
-        protected RequestBuilder(ElasticsearchClient client, ValidateDetectorAction action) {
-            super(client, action, new Request());
```

Comment:
I think most (if not all) empty `Request` constructors existed only for the purpose of being called in `RequestBuilder`s. Would it make sense to get rid of those empty constructors now?

Reply:
Thanks @przemekwitek. If those constructors are now not used then yes, I think now is the time to remove them.